affected:
- package:
    ecosystem: PyPI
    name: pypatchelf
    purl: pkg:pypi/pypatchelf
  ranges:
  - events:
    - introduced: '0'
    type: ECOSYSTEM
  versions:
  - '0.8'
  - '0.9'
aliases:
- CVE-2022-44940
- GHSA-5pcj-3m26-w633
details: Patchelf v0.9 was discovered to contain an out-of-bounds read via the function
  modifyRPath at src/patchelf.cc.
id: PYSEC-2022-43151
modified: '2024-11-25T22:09:33.909779Z'
published: '2022-12-19T22:15:00Z'
references:
- type: EVIDENCE
  url: https://github.com/NixOS/patchelf/pull/419
- type: FIX
  url: https://github.com/NixOS/patchelf/pull/419
- type: WEB
  url: https://github.com/NixOS/patchelf/pull/419
severity:
- score: CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:H
  type: CVSS_V3
